composer.json

```diff
@@ -23,16 +23,15 @@
     "conserto/pomm-symfony-bridge":  "^4.0",
     "symfony/framework-bundle": "~5.0|~6.0",
     "twig/twig": "~3.0",
-    "sensio/framework-extra-bundle": "~5.0|~6.0"
+    "symfony/http-kernel": "^6.2"
```

Comment:
A voir pour conserver la compatibilité sf 5

Reply:
good point. I've changed the target branch to 5.0, because there is some breaking changes (Removal of ParamConverter)